Client/Ecosia/L10N/en.lproj/Ecosia.strings

```diff
@@ -144,7 +144,7 @@
 "The greenest way to search" = "The greenest way to search";
 "Ecosia is the world's most planet-friendly way to search - and it's free." = "Ecosia is the world's most planet-friendly way to search - and it's free.";
 "100% of profits for the planet" = "100% of profits for the planet";
-"We use all our profits for climate action, such as planting trees and generating solar energy." = "We use all our profits for climate action, such as planting trees and generating solar energy.";
+"All our profits go directly to reforestation and renewable energy around the world." = "We use all our profits for climate action, such as planting trees and generating solar energy.";
```

Comment:
Correct. Transifex will find the match for EN and update the value so no action from us is needed 👍

Reply:
But do we want `"We use all our profits for climate action, such as planting trees and generating solar energy."` to be the text that appears on the app? Because I believe transifex will keep it if only the key changes 🤔 Or do you plan to update that in Transifex dashboard?

Reply:
Transifex will update all the keys, find the match for the new given key, update the source (English included) and send it back to us cumulatively in a single PR.

Reply:
Is Transifex automatically uses the key as the value then it makes to leave it the work 😁 I was just not expecting that to be the case since sometimes we want to have the key different than the value